distribution/src/conf/log4j2.properties

```diff
@@ -283,6 +300,10 @@ logger.AUDIT_LOG.level = INFO
 logger.AUDIT_LOG.appenderRef.AUDIT_LOGFILE.ref = AUDIT_LOGFILE
 logger.AUDIT_LOG.additivity = false
 
+logger.opentelemetry-trace.name = tracer
+logger.opentelemetry-trace.level = TRACE
+logger.opentelemetry-trace.appenderRef.OPEN_TELEMETRY.ref = OPEN_TELEMETRY
+
```

Comment:
add logger.opentelemetry.additivity = false

Reply:
Noted and resolved.